tests/unit_tests/kernel_llvm_source_builder_test.py

```diff
@@ -9,25 +9,22 @@
 import pytest
 import os
 
-
-versions = ("kernel/linux-3.10", "kernel/linux-3.10.0-957.el7")
+kernel_dir = "kernel/linux-3.10.0-957.el7"
 
 
 @pytest.fixture
-def builder(request):
+def builder():
     """
     Create kernel source builder that is shared among tests.
-    Parametrized by kernel directory.
     """
-    b = KernelLlvmSourceBuilder(request.param)
+    b = KernelLlvmSourceBuilder(kernel_dir)
     yield b
     b.finalize()
 
 
-@pytest.mark.parametrize("kernel_dir", versions)
-def test_create_kernel(kernel_dir):
+def test_create_kernel():
     """Creating kernel source builder."""
-    builder = KernelLlvmSourceBuilder(kernel_dir)
+    builder = KernelLlvmSourceBuilder("kernel/linux-3.10.0-957.el7")
```

Comment:
What is the purpose of this change? As far as I can tell, leaving `kernel_dir` as the argument to `KernelLlvmSourceBuilder` would be the same but we would have one less hardcoded string.

Reply:
Good catch, fixed.